spec/workers/plos_details_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe PlosDetails do
+  let(:plos_paper) do
+    create(:plos_paper, id: 1, reader: 0, doi: "10.1371/journal.pone.0089204")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.